hook/hook.go

```diff
@@ -25,12 +25,15 @@ type Event struct {
 	TaskInfo mesosutils.TaskInfo
 }
 
+// Env is a container for os.Environ style list of combined environment variable strings.
+type Env []string
```

Comment:
I think this should not be in this package

Reply:
Suggest another then 😄